src/main/java/org/takes/facets/auth/PsCookie.java

```diff
@@ -124,8 +126,8 @@ public Response exit(final Response res,
                 Locale.ENGLISH,
                 "Expires=%1$ta, %1$td %1$tb %1$tY %1$tT GMT",
                 new Date(
-                    System.currentTimeMillis()
-                        + TimeUnit.DAYS.toMillis(this.age)
+                    millis + TimeUnit.DAYS.toMillis(this.age)
```

Comment:
@danff why you dont want to use same method as in tests ?

it can be a constant
```
final SimpleDateFormat format = new SimpleDateFormat(
            "'Expires='EEE, dd MMM yyyy HH:mm:ss 'GMT'"
            Locale.ENGLISH
        );
```
and set the time zone in a static initializer
```
format.setTimeZone(TimeZone.getTimeZone("GMT"));
```

so we can remove once used variable `millis` and make the expression simplier